training/trillium/MAXTEXT_README.md

```diff
@@ -0,0 +1,30 @@
+# Prep for Maxtext GPT3-175B workloads on GKE
```

Comment:
is this specific to GPT3-175? Seems like it applies to the other recipes, so should we delete "GPT3-175B" from this line?